xml_converter/src/xml_converter.cpp

```diff
@@ -121,15 +115,13 @@ void write_protobuf_file(string proto_filepath, map<string, Category>* marker_ca
         cout << "Error making ./protobins" << endl;
         throw std::error_code();
     }
-    // Converts map<string, Category> to a Waypoint::Category
+    // Creates a Waypoint message that contains all categories
```

Comment:
This does not quite describe the next line, but more the next block of code. Maybe change this to:

"Creates a Waypoint message that will be filled with all categories"